deb/preinst

```diff
@@ -11,6 +11,10 @@ if [[ "$1" == "install" || "$1" == "upgrade" ]]; then
     if [  -e /usr/bin/harvest ]; then
       unlink /usr/bin/harvest && echo " --> delete link"
     fi
+    if [ -e /opt/harvest/harvest.yml ]; then
```

Comment:
I don't think we need to backup - in the case that the file is already there - copy the new `harvest.yml` from the deb to `harvest.example.yml`. It's OK if we overwrite `harvest.example.yml`

Reply:
as per my testing, whole /opt/harvest directory gets replaced during rpm upgrade hence in pre install , i had to take backup. All of the backup stuff is to avoid overriding of files during install phase.

Reply:
yuck, ok